conans/test/utils/cpp_test_files.py

```diff
@@ -34,7 +34,7 @@ def build(self):
 
                 command = ('{} && cl /EHsc main.cpp hello{}.lib {}'.format(vcvars, self.name, flags))
                 self.run(command)
-        elif self.settings.compiler == "gcc" and self.settings.os == "Linux":
+        elif tools.os_info.bash_path() and (tools.which("aclocal") or tools.which("aclocal.exe")):
```

Comment:
SHouldn't the ``tools.which`` found it even if you don't add the .exe?

Reply:
I searched for the original PR, I thought there was some better reason to keep it like this, but probably yes, in windows should auto-try the extensions in the same order Windows acts.

https://github.com/conan-io/conan/pull/1801

Reply:
Will take a look